streams.json

```diff
@@ -495,46 +635,41 @@
               "sergey_beryozkin"
           ],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/jettison-json/jettison",
+          "repository_url": "https://github.com/jettison-json/jettison.git",
           "codebase": "master",
           "tag": "jettison-1.3.8",
+          "group_id": "org.codehaus.jettison",
+          "artifact_id": "jettison",
           "version": "1.3.8",
-          "gav": "org.codehaus.jettison",
+          "gav": "jettison:org.codehaus.jettison:1.3.8",
           "comment": ""
         },
         {
           "component_name": "codehaus-woodstox-git",
```

Comment:
It seems it is no longer used after 7.0.z. I would suggest leave it out of the final list in those streams

Reply:
FYI there is https://projects.engineering.redhat.com/browse/SET-44
since EAP 7.1.0, codehaus-woodstox-git is renamed to FasterXML-woodstox 

I agree to remove obsoleted or redundant components from stream it belongs to.

Reply:
Good to know, we are already adding FasterXML-woodstox for those streams, I will remove codehaus-woodstox-git from the affected streams